### ISSUE TITLE
Add mobile-responsive design

### ISSUE THREAD

**Issue by cmccormack** [cmccormack/minesweeper#8]
Site should be mobile responsive from the beginning.  This issue can track any issues with the mobile version of the site.

**Comment by bradtaniguchi** [comment]
I assume this means the styling needs to just "scale" with the UI right? Would this mean having % of the screen for all elements of the game work out, or something else? My CSS skills aren't very strong so I'm curious as to what the best approach here would be :)

**Comment by cmccormack** [comment]
@bradtaniguchi yeah basically restructuring certain elements to better fit on a mobile device, removing unnecessary UI elements that aren't needed unless on desktop, making some text easier to read, surfacing items in dropdown to make it easier to find, etc...
